### ISSUE TITLE
Request to have mods removed

### ISSUE THREAD

**Issue by dmodoomsirius** [BBM-Modpacks/Voltz-Configs#6]
Moved here since the old repo is now private.

**Comment by Arzte** [comment]
k

**Comment by ChatFawkes** [comment]
Thermal Expansion:
Because EnderIO can more or less do everything TE can and personally I think it's just redundant in the modpack. The only thing not covered is the ability to lock machines, but I think those type of security features are bit overpowered for Voltz because players would be more inclined to keep all their goods in those secure containers and then never be able to raid good loot from others.

Thoughts?

**Comment by Arzte** [comment]
I agree with @ChatFawkes on all points, I vote for TE to be removed.

EDIT: I need to improve my git skills.
